controllers/topics/index.js

```diff
@@ -247,15 +250,22 @@ const followTopicV2 = async (req, res) => {
   }
 };
 
-const _afterPutTopic = async (isUnfollow, token, userId, name, isAnonymous) => {
+const _afterPutTopic = async (
+  isUnfollow,
+  token,
+  userId,
+  name,
+  isAnonymous,
+  withSystemMessage = false
+) => {
   // follow / unfollow main feed topic
   try {
     if (isUnfollow) {
       await unfollowMainFeedTopic(token, userId, name);
       await removeTopicFromChatTab(token, name, userId);
     } else {
       await followMainFeedTopic(token, userId, name);
-      await addTopicToChatTab(token, name, userId, isAnonymous);
+      await addTopicToChatTab(token, name, userId, isAnonymous, withSystemMessage);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `_afterPutTopic` function signature has been updated to include a `withSystemMessage` parameter. The function now passes this parameter to the `addTopicToChatTab` function. This change seems consistent with the rest of the changes in this PR.
```diff
-const _afterPutTopic = async (isUnfollow, token, userId, name, isAnonymous) => {
+const _afterPutTopic = async (
+  isUnfollow,
+  token,
+  userId,
+  name,
+  isAnonymous,
+  withSystemMessage = false
+) => {
   // follow / unfollow main feed topic
   try {
     if (isUnfollow) {
       await unfollowMainFeedTopic(token, userId, name);
       await removeTopicFromChatTab(token, name, userId);
     } else {
       await followMainFeedTopic(token, userId, name);
-      await addTopicToChatTab(token, name, userId, isAnonymous);
+      await addTopicToChatTab(token, name, userId, isAnonymous, withSystemMessage);
     }
   } catch (error) {
     console.log('After put topic error: ', error);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->